build/patches/Partition-DNS-requests-by-top-frame-NIK.patch

```diff
@@ -361,16 +361,16 @@ diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
          resolve_context_->isolation_info(), request_priority_);
      ++attempts_count_;
      int rv = attempts_.back()->Start(base::BindOnce(
-@@ -1703,6 +1716,8 @@ class DnsTransactionImpl : public DnsTransaction,
+@@ -1689,6 +1702,8 @@ class DnsTransactionImpl : public DnsTransaction,
    base::SafeRef<ResolveContext> resolve_context_;
-   RequestPriority request_priority_;
+   RequestPriority request_priority_ = DEFAULT_PRIORITY;
  
 +  const NetworkIsolationKey& network_isolation_key_;
```

Comment:
this is more right like
```
const NetworkIsolationKey network_isolation_key_;
```
the entire chromium code handles network isolation as copy rather than by reference.
it was something I hadn't noticed before.

Reply:
What is the impact? The patch is ineffective right now?

Reply:
I see it used also with the reference elsewhere.

Reply:
> What is the impact? 

Prevents the destruction of the object to which it refers until the destruction of the dns request.
In our case no impact, just style, chromium seems to prefer that way.

> The patch is ineffective right now?

no, the patch still works

> I see it used also with the reference elsewhere.

as an object variable there is no other point